kolibri/core/assets/src/vue/exercise-attempts/answer-icon.vue

```diff
@@ -1,9 +1,9 @@
 <template>
 
   <div>
-    <svg v-if="isRight && !isHint" src="./icons/right.svg" class="yes" :class="{ yay: success }"></svg>
-    <svg v-if="!isRight && !isHint" src="./icons/wrong.svg" class="no" :class="{ yay: success }"></svg>
-    <svg v-if="isHint" src="./icons/hint.svg" class="no" :class="{ yay: success }"></svg>
+    <svg      v-if="answer === 'right'" src="./icons/right.svg" class="yes" :class="{ yay: success }"/>
+    <svg v-else-if="answer === 'wrong'" src="./icons/wrong.svg" class="no"  :class="{ yay: success }"/>
+    <svg v-else-if="answer === 'hint'"  src="./icons/hint.svg"  class="no"  :class="{ yay: success }"/>
```

Comment:
I used a computed value to set something like this up inside of `user-edit-modal` and I thought it came out really well. Might not be your code, but just wanted to throw that out there.